api/MouseEvent.json

```diff
@@ -856,7 +856,7 @@
               }
             ],
             "edge": {
-              "version_added": true
+              "version_added": "13"
```

Comment:
`document.createEvent('MouseEvent').movementX` confirmed missing in Edge 12 but present in Edge 18, so I trust this.